src/types/widepixel.jl

```diff
@@ -0,0 +1,10 @@
+struct WidePixel{C <: Colorant, X <: Tuple}
+    color::C
+    extra::X
+end
+
+Base.zero(::WidePixel{C, X}) where {C, X} = WidePixel(zero(C), zero.(fieldtypes(X)))
+
+function Base.show(io::IO, x::WidePixel{C,X}) where {C, X}
+    print(io, "WidePixel($(x.color))")
```

Comment:
Brevity. There's one color type, whereas extra channels are unbounded

Reply:
Maybe add text saying "and N extra channels" where N is the number of channels.

Reply:
It's already crowded (even moreso for RGB images)

```
256×256 TiffImages.DenseTaggedImage{TiffImages.WidePixel{ColorTypes.Gray{Float...
 WidePixel(Gray{Float32}(0.09607481f0))   WidePixel(Gray{Float32}(0.09607481f0)
 WidePixel(Gray{Float32}(0.09607481f0))   WidePixel(Gray{Float32}(0.09607481f0)
 WidePixel(Gray{Float32}(0.09607481f0))   WidePixel(Gray{Float32}(0.09607481f0)
 WidePixel(Gray{Float32}(0.09607481f0))   WidePixel(Gray{Float32}(0.09607481f0)
 WidePixel(Gray{Float32}(0.0963287f0))    WidePixel(Gray{Float32}(0.0963287f0))
 WidePixel(Gray{Float32}(0.0963287f0))    WidePixel(Gray{Float32}(0.0963287f0))
 WidePixel(Gray{Float32}(0.0963287f0))    WidePixel(Gray{Float32}(0.0963287f0))
 WidePixel(Gray{Float32}(0.0963287f0))    WidePixel(Gray{Float32}(0.0963287f0))
 WidePixel(Gray{Float32}(0.0963287f0))    WidePixel(Gray{Float32}(0.0963287f0))
 WidePixel(Gray{Float32}(0.0963287f0))    WidePixel(Gray{Float32}(0.0963287f0))
```

Reply:
We should use [`compact=true`](https://docs.julialang.org/en/v1/base/io-network/#Base.IOContext-Tuple{IO,%20Pair}) to change the printing style for arrays and have a more verbose option otherwise.